### ISSUE TITLE
About Search Time 

### ISSUE THREAD

**Issue by zkjst110** [ianwhale/nsga-net#7]
In the paper

> We set the number of phases np to three and the number of nodes in each phase no to six.
> Duringarchitecturesearch, we limit the number of filters (channels) in any node to 16 for each
> one of the generated network architecture. We then train them on our training set using standard stochastic gradient descent (SGD) back-propagation algorithm and a cosine annealing learning rate schedule. Our initial learning rate is 0.025 and we train for 25 epochs, which takes about 9 minutes on a NVIDIA 1080Ti GPU implementation in PyTorch .

But in the code, the default parameter --n_nodes(number of nodes per phases) is four.
I set the channel to 16 and n_nodes to six ,but the search process is slow. So, I want to know if you have the concrete configuration about the 9 minutes.
Also , I find that the search code run slower in multiple GPU than run in a GPU,can you explain the phenomenon？
Tnank you very much!
